clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -2405,9 +2408,12 @@ protected long getTopicPartitionEndOffSet(String kafkaUrl, PubSubTopic pubSubTop
           Duration.ofMillis(10),
           Duration.ofMillis(500),
           Duration.ofSeconds(5),
-          Collections.singletonList(VeniceException.class));
+          Arrays.asList(PubSubTopicDoesNotExistException.class, PubSubOpTimeoutException.class, VeniceException.class));
+    } catch (PubSubTopicDoesNotExistException | PubSubOpTimeoutException e) {
+      LOGGER.error("Failed to get end offset for topic-partition: {} even after 10 retries", topicPartition, e);
+      return StatsErrorCode.LAG_MEASUREMENT_FAILURE.code;
     } catch (Exception e) {
-      LOGGER.error("Could not find latest offset for {} even after 5 retries", pubSubTopic.getName());
+      LOGGER.error("Could not find latest offset for {} even after 10 retries", pubSubTopic.getName());
```

Comment:
let's log PubSubTopicPartition

Besides timeouts are pretty tight on this API IMHO